lib/pure/includes/osenv.nim

```diff
@@ -5,9 +5,10 @@ when not declared(os) and not declared(ospaths):
 
 when defined(nodejs):
   proc getEnv*(key: string, default = ""): string {.tags: [ReadEnvEffect].} =
-    var ret: cstring
+    var ret = default.cstring
     let key2 = key.cstring
-    {.emit: "`ret` = process.env[`key2`];".}
+    {.emit: "const value = process.env[`key2`];".}
```

Comment:
pre-existing but `asm "const value = process.env[`key2`];"` leads to nicer syntax

Reply:
Assuming not critically relevant, generates exactly the same code for me... :shrug: